daemon/src/taker_cfd.rs

```diff
@@ -167,6 +189,71 @@ impl<O, M, W> Actor<O, M, W> {
 
         Ok(())
     }
+
+    async fn handle_auto_rollover(&mut self) -> Result<()> {
+        tracing::debug!("Auto rollover");
+
+        let mut conn = self.db.acquire().await?;
+        let cfds = load_all_cfds(&mut conn).await?;
+
+        // cleanup all pending proposals because auto-rollover will create a new one
+        self.current_pending_proposals.clear();
+
+        // Only cfds in state `Open` are allowed for auto-rollover proposal
+        for cfd in cfds
+            .iter()
+            .filter(|cfd| matches!(cfd.state, CfdState::Open { .. }))
+        {
+            tracing::debug!(
+                "settlement_interval: {:?}",
```

Comment:
Debug formatting a duration is going to be noisy. Please either do seconds or something like this: https://docs.rs/humantime/2.1.0/humantime/fn.format_duration.html

Reply:
removing, i don't think we need to print this anymore